copulas/univariate/gaussian.py

```diff
@@ -44,11 +44,13 @@ def fit(self, X):
 
         if isinstance(X, (pd.Series, pd.DataFrame)):
             self.name = X.name
-        else:
-            self.name = None
 
-        self.mean = np.mean(X)
-        self.std = np.std(X) or 0.001
+        self.constant_value = self._get_constant_value(X)
+
+        if not self.constant_value:
```

Comment:
Remember that `bool(0) == False`.
This should be an explicit check for `None`:  `if self.constant_value is not None`